mut/stage.py

```diff
@@ -84,6 +85,30 @@
 T = TypeVar('T')
 
 
+class CacheControl:
```

Comment:
[nit] Maybe worth calling this class `GatsbyCacheControl` since it only implements Gatsby-specific rules?

Reply:
This is used for both legacy and gatsby